### ISSUE TITLE
Removing all homepage subreddits causes Slide to re-add all subreddits

### ISSUE THREAD

**Issue by jimbo1qaz** [ccrama/Slide#3046]
Slide 6.2 from F-Droid.

- It's difficult to remove all subreddits from the homepage and sidebar. In the settings, I have to long-press one, then tap the remaining ones one by one.
- Once I remove all subreddits from the settings, the program adds the entire default set back.

I want to start from a clean slate and add only subreddits i choose, to keep my home page tabs clean.

**Comment by Tensounder54** [comment]
Yep, can confirm this is still a big on 6.2.2. Thanks for bringing this up. We will look in to it.

**Comment by goforbg** [comment]
Hmm, it'd be easy to set a SharedPreference to set to over18=true after the first time the prompt is shown for a subreddit, but then it seems like the pain point is to prompt the user to sign in, so I need a better idea if this can be accepted here.

**Comment by nyanpasu64** [comment]
@goforbg did you post to the wrong issue?

**Comment by goforbg** [comment]
> @goforbg did you post to the wrong issue?

Yes I'm sorry, I should have posted that on the next issue.

**Comment by aliraad79** [comment]
Hi can i work on this?
